### PR TITLE
Added color blind friendly dark theme

### PR DESCRIPTION
**Pull request by alexndrvega**
The `tittle_color` and `icon_color` are color blind friendly and detectable by people with protanopia, deuteranopia, and tritanopia.

Theme: `color-blind-friendly-dark`

GitHub stats

Screen Shot 2020-08-08 at 9 31 00 PM

### CODE REVIEW

File: themes/index.js [hunk 1]
```diff
@@ -167,6 +167,12 @@ const themes = {
     text_color: "24292e",
     bg_color: "ffffff",
   },
+  "color-blind-friendly-dark": {
```
Comment (anuraghazra) [inline themes/index.js:170]: can we change the name of the theme to something shorter? this is too long. 

"a11y-dark", "vision-friendly-dark" ?

Comment (alexndrvega) [reply themes/index.js]: I have changed the theme name to `vision-friendly-dark`

**Comment by anuraghazra** [comment]
Looks good! Thanks @alexndrvega

**Comment by anuraghazra** [comment]
Theme preview (bot)

**Comment by anuraghazra** [comment]
Theme preview (bot)

**Pull request merged**